### PR TITLE
Final Pull Request

### PR DESCRIPTION
**Pull request by mistermaxis**
### Tasks Completed:
- Create basic structure for Missions - Setup
- Fetch missions  - Fetch data
- Fetch rockets - Fetch data
- Display missions - List render
- Display rockets - List render
- Implement rocket booking - Actions
- Implement mission joining - Actions
- Implement rocket booking cancellation - Actions
- Implement mission leaving - Actions
- Switch badges for Missions - Conditional components
- Switch badges for Rockets - Conditional components
- Display reserved rockets in My profile - Filters
- Display joined missions in My profile - Filters

### CODE REVIEW

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @mistermaxis & @ixboy, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark:  Good commit messages
:heavy_check_mark: Descriptive Readme file
:heavy_check_mark: The project is running as it should

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments below and under the review._

- [ ] As it is a requirement for this project and best practice, please kindly add tests to your project, this will ensure your app does break when you make changes. See screenshot of the project requirement below and kindly fix that. 

team1

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

- [OPTIONAL] Directories for all Redux state slice files should be created (rockets, missions). It would be nice to implement that as it makes your code more organized.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/missions/missions.test.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    {component}
+  </Provider>,
+);
+
+describe('Testing react testing library', () => {
+  it('Renders react components correctly', async () => {
+    render(<MissionList />);
+
+    const buttons = await screen.findAllByText('Join Mission');
+    const badges = await screen.findAllByText('NOT A MEMBER');
```
Comment (ArthurGC) [inline src/components/missions/missions.test.js:18]: - [ ] Please, It would be better if you fix your test because there are failing. ♻️ 

test-space

**Review by ArthurGC — CHANGES_REQUESTED** [review]
# Required Changes ♻️ :warning: 
Hi  Team

Great work on making the changes requested by a previous reviewer. 👏 
You've done well implementing some of the requested changes but there are some issues that you still need to work on to go to the next project but you are almost there! 



## Good Points :+1: :
Great job following the first reviewer's suggestions :heavy_check_mark:

## Aspects to improve ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi Team 🙋 , 

Please work on the changes requested by a previous reviewer 👏 
The tests are still failing 


 # Required Changes ♻️ 🚧 
_Check the comments under the review._

### Optional Suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by mistermaxis** [comment]
test

> I'm sorry but they are working on my machine.

**Review by dasileker — APPROVED** [review]
## **Hi @mistermaxis,** 

### **Awesome Work Team 👏👌**

## **Congratulations! 🎉 Project Approved ✅**
Your project is complete!  There is nothing else to say other than... it's time to merge it 🚀 
 
Approved


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**